src/App.jsx

```diff
@@ -14,70 +15,58 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const defaultSelect = 'No goods selected';
+  const firstSelected = 'Jam';
+  const [value, setValue] = useState(firstSelected);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      {value === defaultSelect ? (
+        <h1 className="title is-flex is-align-items-center">
+          {defaultSelect}
+        </h1>
+      ) : (
+        <h1 className="title is-flex is-align-items-center">
+          {`${value} is selected`}
+          <button
+            onClick={() => {
+              setValue(defaultSelect);
+            }}
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+          />
+        </h1>
+      )}
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
-
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Garlic
-          </td>
-        </tr>
-      </tbody>
-    </table>
-  </main>
-);
+      <table className="table">
+        <tbody>
+          {goods.map((good, index) => (
+            <tr data-cy="Good" key={`${index + 1}`} className={value === good ? 'has-background-success-light' : ''}>
```

Comment:
```suggestion
          {goods.map((good, index) => {
            const isSelectedGood = value === good;

            return (
              <tr data-cy="Good" key={`${index + 1}`} className={value === good ? 'has-background-success-light' : ''}>
```
and reuse this value everywhere you need